native/dtrevc3.go

```diff
@@ -0,0 +1,861 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package native
+
+import (
+	"math"
+
+	"github.com/gonum/blas"
+	"github.com/gonum/blas/blas64"
+	"github.com/gonum/lapack"
+)
+
+// Dtrevc3 computes some or all of the right and/or left eigenvectors of an n×n
+// upper quasi-triangular matrix T in Schur canonical form. Matrices of this
+// type are produced by the Schur factorization of a real general matrix A
+//  A = Q T Q^T,
+// as computed by Dhseqr.
+//
+// The right eigenvector x of T corresponding to an
+// eigenvalue λ is defined by
+//  T x = λ x,
+// and the left eigenvector is defined by
+//  y^H T = λ y^H.
```

Comment:
Reference lapack says just "transpose" here, but it should say "conjugate transpose" (the code confirms it).